src/packages/migrate/src/utils/seed.ts

```diff
@@ -172,6 +173,7 @@ To install them run: ${chalk.green(
       // User can customize the `ts-node` command from the package script
       if (scripts?.['ts-node']) {
         tsNodeCommand = scripts['ts-node']
+        // TODO Why does this not need getSeedScript?
```

Comment:
Because it's "safer" and you can pass any command there.

Anyway, we are currently looking at renaming and changing the behavior here.

Reply:
Just for my understanding as I am curious: What makes running the file via the `ts-node` command (Whatever it may be) safer than doing the standard `ts-node --eval`?

Reply:
`--eval` makes it possible to run this code without having it on the filesystem which was practical for our code snippet doing the execution of export.

Running this snippet with the custom ts-node command means we don't control the command completely and I could see problems depending on the user command input here.